horizons/ext/speaklater.py

```diff
@@ -200,7 +200,7 @@ def __repr__(self):
         try:
             return 'l' + repr(self.value)
         except Exception:
-            return '<%s broken>' % self.__class__.__name__
```

Comment:
This is an external library, and therefore shouldn't be modified as it doesn't fall under our codestyle. This change should be reverted.